### PR TITLE
add relative_require for other timers

### LINKED ISSUE #129: uninitialized constant Circuitbox::Timer::Null (NameError)
I'm getting this error on the latest pre version(2.0.0.pre1). It would seem that require_relative is missing for the other Timer classes.

### PR DESCRIPTION
**Pull request by ritikesh**
Fixes https://github.com/yammer/circuitbox/issues/129

### CODE REVIEW

**Comment by ritikesh** [comment]
is there an update on this?

File: lib/circuitbox/circuit_breaker.rb [hunk 1]
```diff
@@ -78,12 +78,10 @@ def run(circuitbox_exceptions: true)
...
 
     def open?
-      if open_flag?
-        true
-      else
-        false
-      end
+      circuit_store.key?(open_storage_key)
     end
+    alias :open_flag? :open?
```
Comment (matthewshafer) [inline lib/circuitbox/circuit_breaker.rb]: Is there a reason for keeping ```open_flag?``` around besides not having to rename the ```open_flag?``` calls to ```open?```?

Comment (ritikesh) [reply lib/circuitbox/circuit_breaker.rb]: Since this is a private method, I don't see a reason why we shouldn't. Updated, pls have a look.

Comment (matthewshafer) [reply lib/circuitbox/circuit_breaker.rb]: Thanks for updating. Sorry it took so long for me to get to this, my notification preferences on this repo were removed.

Comment (ritikesh) [reply lib/circuitbox/circuit_breaker.rb]: Thanks for merging! Do you know anybody I can reach out to for this(https://github.com/yammer/yam/pull/54)? This one's been open for a while as well.

Comment (Sija) [reply lib/circuitbox/circuit_breaker.rb]: @matthewshafer Any chances for a new release soon?

Comment (matthewshafer) [reply lib/circuitbox/circuit_breaker.rb]: @ritikesh I'll ask around.

@Sija I'll get out another pre this week. I won't have documentation and upgrade guide done this week so it won't be a final release.

Comment (ritikesh) [reply lib/circuitbox/circuit_breaker.rb]: Thanks. 👍

Comment (Sija) [reply lib/circuitbox/circuit_breaker.rb]: @matthewshafer Doesn't matter as long it's published on RubyGems :)

Comment (matthewshafer) [reply lib/circuitbox/circuit_breaker.rb]: @sija 2.0.0.pre2 is on rubygems, let me know if you run into any issues.

Comment (Sija) [reply lib/circuitbox/circuit_breaker.rb]: @matthewshafer Works like a charm, thanks!

**Pull request merged**